### DISCUSSION TITLE
Automatic checkpoint when writing/deleting

### DISCUSSION THREAD

**Discussion by simonaubertbd** [duckdb/duckdb#8796] [Ideas]
Hello,

I would like a setting (maybe even a default setting??) that allows to do automatically the checkpoint when writing (create, insert, alter...) or deleting (drop, delete...) data. I hate to do it manually and it would help interoperability whith other platforms.

Best regards,

Simon

**Comment by fanvanzh** [comment]
set wal_autocheckpoint to '16MB';
